[16.0][ADD] shopinvader_sale_cart_no_onchange_helper module (#1593)
shopinvader/odoo-shopinvader: shopinvader_sale_cart_no_onchange_helper/readme/DESCRIPTION.rst

> Remove onchange_helper from shopinvader_sale_cart module

Reviewer: ```suggestion
Disables the use of onchange_helper in shopinvader_sale_cart
```
